package.json

```diff
@@ -26,7 +26,7 @@
     "prepare": "npm run build",
     "prepublishOnly": "npm test && npm run lint",
     "graphql:codegen": "graphql-codegen --config graphql-codegen.yml",
-    "swagger:codegen": " openapi --input https://raw.githubusercontent.com/cowprotocol/services/main/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false"
+    "swagger:codegen": "openapi --input https://raw.githubusercontent.com/cowprotocol/services/main/crates/orderbook/openapi.yml --output src/order-book/generated --exportServices false --exportCore false --useUnionTypes"
```

Comment:
This is the main change. See what the option does [in the docs](https://github.com/ferdikoomen/openapi-typescript-codegen/blob/HEAD/docs/enum-vs-union-types.md)

Everything else is adapting to the consequences of it.